src/System.Windows.Forms/tests/InteropTests/InteropTestBase.cs

```diff
@@ -0,0 +1,22 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit.Sdk;
+
+namespace System.Windows.Forms.Tests
+{
+    public class InteropTestBase
```

Comment:
Can we turn this into an abstract?
```suggestion
    public abstract class InteropTestBase
```